more/free-programming-interactive-tutorials-en.md

```diff
@@ -313,6 +319,7 @@
 * [SQL at Codecademy](https://www.codecademy.com/courses/learn-sql)
 * [SQL Tutorial](https://www.w3schools.com/sql) - W3Schools
 * [SQLBolt](http://sqlbolt.com)
+* [SQL Server Tutorial](https://www.tutlane.com/tutorial/sql-server)
```

Comment:
- Add creator to disambiguate resources
- Fix Travis linting (https://app.travis-ci.com/github/EbookFoundation/free-programming-books/builds/225315532#L225)

```suggestion
* [SQL at Codecademy](https://www.codecademy.com/courses/learn-sql)
* [SQL Server Tutorial](https://www.tutlane.com/tutorial/sql-server) - tutlane
* [SQL Tutorial](https://www.w3schools.com/sql) - W3Schools
* [SQLBolt](http://sqlbolt.com)
```